competency-explorer/dist/bundle.js

```diff
@@ -264,6 +264,1098 @@ eval("module.exports = __webpack_require__(/*! regenerator-runtime */ \"./node_m
 
 /***/ }),
```

Comment:
because of the way github pages are hosted, we need to include the `dist` folder here.